_posts/2022-01-27-jump_dev_2022.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title:  "JuMP-dev 2022"
+date:   2022-01-18
+categories: [jump-dev]
+permalink: /meetings/juliacon2022/
+---
+
+We are pleased to announce that JuMP-dev 2022 will be co-located with
+[JuliaCon 2022](https://juliacon.org/20212)!
+
+This year JuliaCon is virtual, and will be held 27-29 July, 2022.
+
+## Outline
+
+The purpose of JuMP-dev is to bring together students, researchers, and
+practitioners with interests in the software aspects of
+[JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
+we invite new contributors and those who have not met the core development team.
+
+## How do I attend?
+
+JuliaCon 2022 is entirely virtual and free to attend, but
+[registration is required](https://juliacon.org/2022/tickets/).
+
+All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2022/coc/).
+
+## How do I give a talk?
+
+Similarly to [previous](/meetings/mit2017) [iterations](/meetings/bordeaux2018)
+[of the](/meetings/santiago2019) [workshop](/meetings/juliacon2021), we invite
+participants to present work related to JuMP. In particular, we especially seek
+talks related to:
+
+- JuMP core development ([MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl), JuMP 1.0)
+- Mathematical optimization solvers written in Julia
+- Automatic differentiation in Julia (e.g., Cassette, Zygote)
+- Julia interfaces to solvers
+- JuMP extensions (stochastic programming, robust optimization, multiobjective optimization, ...)
+- Optimization libraries that use JuMP
```

Comment:
Maybe:
 - Software that relies on JuMP
 
 I am thinking of something like PowerModels that might be seen by some as a simulator of power systems rather than optimization.
 
 Or maybe just leave by the end of the list:
  - Other applications that rely on JuMP